go.mod

```diff
@@ -4,11 +4,13 @@ go 1.17
 
 require (
 	github.com/evanphx/json-patch v4.12.0+incompatible
+	github.com/evanphx/json-patch/v5 v5.6.0
```

Comment:
Not sure if we should use two versions of the same library. Maybe we should only use v4.12 for now (like k/k)